app.js

```diff
@@ -0,0 +1,90 @@
+// import functions and grab DOM elements
+
+import { compareNumbers, setGuessesLeftDisplay } from "./utils.js";
```

Comment:
I recommend figuring out the eslint setting that auto corrects stuff like this on save.